spec/models/searchgov_domain_spec.rb

```diff
@@ -302,21 +307,32 @@
       context 'when the redirect is to https' do
         let(:new_url) { 'https://agency.gov/' }
 
-        it 'sets the status to 200' do
-          expect{ check_status }.to change{ searchgov_domain.reload.status }.from(nil).to('200 OK')
+        it 'sets the status to 200 OK' do
+          expect{ check_status }.to change{
+            searchgov_domain.reload.status
```

Comment:
I noticed in https://github.com/rubocop-hq/rspec-style-guide and http://www.betterspecs.org/ they put a space between the brackets. 
expect { check_status ).to change {

Reply:
👍